riscv/csrs.cc

```diff
@@ -996,7 +996,19 @@ reg_t wide_counter_csr_t::read() const noexcept {
 }
 
 void wide_counter_csr_t::bump(const reg_t howmuch) noexcept {
-  val += howmuch;  // to keep log reasonable size, don't log every bump
+  auto config_val = 0;
+
+  if (proc->extension_enabled_const(EXT_SMCNTRPMF)) {
+    if (address == CSR_MCYCLE) {
+      config_val = state->csrmap[CSR_MCYCLECFG].get()->read();
+    } else if (address == CSR_MINSTRET) {
+      config_val = state->csrmap[CSR_MINSTRETCFG].get()->read();
+    }
+  }
+
+  if (!is_counting_inhibited(config_val)) {
```

Comment:
Should the method be `is_counting_enabled()` instead, to avoid the double negative here?

Reply:
Done.